bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -71,11 +80,15 @@
     final long ledgerId;
     long lastAddPushed;
     volatile long lastAddConfirmed;
+    volatile long explicitLastAddConfirmed;
```

Comment:
If we put the flushing lac logic into a separated class,

- it is easier to test
- the code will become clear
- if explicitLac is enabled, we instantiate the explicit lac flush policy, otherwise, we create a NULL flush policy.